test/absinthe/schema/sdl_render_test.exs

```diff
@@ -104,7 +104,8 @@ defmodule SdlRenderTest do
   end
 
   test "Render SDL from blueprint defined with SDL" do
-    assert Absinthe.Schema.to_sdl(SdlTestSchema) == SdlTestSchema.sdl()
+    assert Absinthe.Schema.to_sdl(SdlTestSchema, include_disconnected: true) ==
```

Comment:
The SDL in that module has unattached types, so we need to use the option to make the test pass. We could also connect all the types, but I think it's helpful to have a test case showing that things can be preserved.

Reply:
That's just an artifact of the previous situation - I'd be fine tweaking the test such that all of them are "used"...

Reply:
Given your comments elsewhere referencing the spec, I think this makes sense. People can always run the pipeline themselves if they want absolutely every type.